[docs][intro/readme]: add admission simple description (#144)
hwameistor/hwameistor: README.md

### Scheduler

> The Scheduler is to automatically schedule the Pod to the correct node which has the associated HwameiStor volume.

Reviewer: ```suggestion
Scheduler is to automatically schedule a pod to a correct node which has the associated HwameiStor volumes.
```

> ### Admission-Controller

Reviewer: ```suggestion
### admission-controller
```

> Admission-Controller is a webhook that can automatically determine which pod uses the HwameiStor volume and, help to modify the schedulerName to hwameistor-scheduler.

Reviewer: ```suggestion
admission-controller is a webhook that can automatically determine which pod uses the HwameiStor volume and, help to modify the schedulerName to hwameistor-scheduler.
```

> Admission-Controller is a webhook that can automatically determine the pod that uses the HwameiStor Volume and help it modify the schedulerName to hwameistor-scheduler.

Reviewer: ```suggestion
Admission-Controller is a webhook that can automatically determine which pod uses the HwameiStor volume and, help to modify the schedulerName to hwameistor-scheduler.
```
Could we give a consistent word for `Admission-Controller`. Otherwise, it may contains many variants later that will confuse most people:
- admission-controller
- Admission-Controller
- Admission Controller
- Admission controller
Maybe we should define a glossary list.

## Documentation

> Full documentation is hosted at our project website [hwameistor.io](https://hwameistor.io/docs/intro).

Reviewer: ```suggestion
For full documentation see website [hwameistor.io](https://hwameistor.io/docs/intro).
```
